Added config file for cupper with configuration

gnurenga opened the pull request:
Steps to test the new theme

1. clone the theme as submodule 
git submodule add https://github.com/zwbetz-gh/cupper-hugo-theme.git themes/cupper-hugo-theme

2. use `cupper_config.yaml` by running the below command 

```shell

$ hugo server --config cupper_config.yaml
```

3. Cupper theme work only with hugo version > 0.60.1

4. Hugo version I used for testing - v0.74.3

gnurenga committed:
Added config file for cupper with configuration
for the content.

mohan43u commented:
thanks for the MR, noticed below issues

1. Blank Index page, only sidebar and footer appears
2. No way to see blog posts
3. Cupper logo, also Logo caption need to be changed to reflect Ilugc

gnurenga commented:
> 
> 
> thanks for the MR, noticed below issues
> 
>     * Blank Index page, only sidebar and footer appears
>

I have noticed it after pull request. Will fix it. 
 
>     * No way to see blog posts

I will add a path for blog and add it in the menu. 

>     * Cupper logo, also Logo caption need to be changed to reflect Ilugc

I will remove the logo. Do ILUGC have any logo ?

mohan43u commented:
> I will remove the logo. Do ILUGC have any logo ?

I'm not sure, need check with seniors, meanwhile, you can hide the logo and keep only caption.

gnurenga committed:
1. Added logo and credit for the logo in footer
2. cupper config file is changed ,
   * Removed the descriptiong below the logo
   * Added ILUGC expansion below logo
   * Added URL to the logo credit page
   * Menu item is rearanged

gnurenga commented:
@mohan43u 

* I have added a logo which is open sourced. Also added citation to the logo http://ecogex.com/the-other-linux-logo/
* _index.md is added to publish the content in the home page. Since this theme requires slight modification  i have copied about.md to _index.md inside  content/ folder.
* Blog role in the landing page is specific to theme mainroad. Not sure how to change the default layout.

gnurenga commented:
### Landing page preview

![Screenshot from 2020-10-26 10-23-06](https://user-images.githubusercontent.com/3196294/97135130-7830ab00-1775-11eb-9dfb-3dfde7612ff6.png)

mohan43u commented:
Hi,

I see this when I tried, is the css/js/images are added to your PR?

https://drive.google.com/file/d/1tIbZyTj3cXtTsHsFj-b_lT3mHe88FRg0/view?usp=sharing

Also, It will be helpful if you find any theme which did what mainroad do on the landing page, people has to click one more link to land on posts we put in cupper.

gnurenga commented:
> Hi,
> 
> I see this when I tried, is the css/js/images are added to your PR?

Yes, I have added log.svg. But I am not sure why you are getting error. For me the landing page properly loaded. 
 
> https://drive.google.com/file/d/1tIbZyTj3cXtTsHsFj-b_lT3mHe88FRg0/view?usp=sharing
> 
> Also, It will be helpful if you find any theme which did what mainroad do on the landing page, people has to click one more link to land on posts we put in cupper.

One hack we can do, use the main page layout from main road.
